python/packages/isce3/focus/caltone.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from isce3.signal import cola_windows
+
+class ToneRemover:
+    def __init__(self, frequency, n, window_size=64, dtype=np.complex64):
```

Comment:
We discussed this in our review offline.

I think the things that most need to be clarified via docstrings/comments are:

* The `analyze` method is effectively performing a single-bin FFT to get the spectral component of the signal data at the caltone frequency. 
* The `synthesize` method is performing the inverse FFT to get that signal component back in the time domain
* Once we have that signal component in the time domain, we can simply remove it from the original signal data.
* We apply the FFT in blocks because (while the caltone frequency remains constant) the amplitude and phase of the caltone may vary across the range line.